nuxeo-features/nuxeo-elasticsearch/nuxeo-elasticsearch-audit/src/test/java/org/nuxeo/elasticsearch/TestAuditMigration.java

```diff
@@ -103,7 +127,7 @@ public void shouldMigrate() throws Exception {
         final long nbEntriesToMigrate = res.get(0).longValue();
         Assert.assertEquals(1000, nbEntriesToMigrate);
 
-        AuditBackend backend = audit.getBackend();
+        AuditBackend backend = (AuditBackend) Framework.getService(AuditLogger.class);
```

Comment:
Is it normal to ask for an AuditLogger and cast it to an AuditBackend?

Reply:
It was because this interface was originally not provided in the audit service conponent. I'll add it.